tests/conftest.py

```diff
@@ -307,85 +290,281 @@ def simple_inventory_node() -> None:
+    title += "SOft coarse grained Monte-Carlo Acceleration (SOMA)"
+
+    reference = cript.Reference(
+        "journal_article",
+        title=title,
+        authors=["Ludwig Schneider", "Marcus Müller"],
+        journal="Computer Physics Communications",
+        publisher="Elsevier",
+        year=2019,
+        pages=[463, 476],
+        doi="10.1016/j.cpc.2018.08.011",
+        issn="0010-4655",
+        website="https://www.sciencedirect.com/science/article/pii/S0010465518303072",
+    )
+    return reference
+
+
+@pytest.fixture(scope="function")
+def simple_reference_dict() -> dict:
+    ret_dict = {
+        "node": ["Reference"],
+        "type": "journal_article",
+        "title": "Multi-architecture Monte-Carlo (MC) simulation of soft coarse-grained polymeric materials: SOft coarse grained Monte-Carlo Acceleration (SOMA)",
+        "authors": ["Ludwig Schneider", "Marcus Müller"],
+        "journal": "Computer Physics Communications",
+        "publisher": "Elsevier",
+        "year": 2019,
+        "pages": [463, 476],
+        "doi": "10.1016/j.cpc.2018.08.011",
+        "issn": "0010-4655",
+        "website": "https://www.sciencedirect.com/science/article/pii/S0010465518303072",
+    }
+    return ret_dict
 
 
 @pytest.fixture(scope="function")
 def simple_citation_node(simple_reference_node) -> cript.Citation:
-    """
-    minimal citation node
-    """
-    my_citation = cript.Citation(type="derived_from", reference=simple_reference_node)
+    citation = cript.Citation("reference", simple_reference_node)
+    return citation
+
 
-    return my_citation
+@pytest.fixture(scope="function")
+def simple_citation_dict(simple_reference_dict) -> dict:
+    ret_dict = {"node": ["Citation"], "reference": simple_reference_dict, "type": "reference"}
+    return ret_dict
 
 
 @pytest.fixture(scope="function")
 def simple_quantity_node() -> cript.Quantity:
-    """
-    minimal quantity node
-    """
-    my_quantity = cript.Quantity(key="mass", value=1.23, unit="gram")
+    quantity = cript.Quantity("mass", 11.2, "kg", 0.2, "std")
+    return quantity
 
-    return my_quantity
+
+@pytest.fixture(scope="function")
+def simple_quantity_dict() -> dict:
+    ret_dict = {"node": ["Quantity"], "key": "mass", "value": 11.2, "unit": "kg", "uncertainty": 0.2, "uncertainty_type": "std"}
+    return ret_dict
+
+
+@pytest.fixture(scope="function")
+def simple_software_node() -> cript.Software:
+    software = cript.Software("SOMA", "0.7.0", "https://gitlab.com/InnocentBug/SOMA")
+    return software
+
+
+@pytest.fixture(scope="function")
+def simple_software_dict() -> dict:
+    ret_dict = {"node": ["Software"], "name": "SOMA", "version": "0.7.0", "source": "https://gitlab.com/InnocentBug/SOMA"}
+    return ret_dict
+
+
+@pytest.fixture(scope="function")
+def simple_property_node(simple_material_node, simple_condition_node, simple_citation_node, simple_data_node, simple_process_node, simple_computation_node):
+    p = cript.Property(
```

Comment:
`p` is a very bad variable name, variable names should be self documenting

Reply:
I know, but I don't care enough for tests to fix it.

Reply:
If you actually wanted to prevent this, you can set up pylint with trunk to sort out for bad variable names.